controllers/secret_controller.go

```diff
@@ -147,8 +147,6 @@ type SecretReconciler struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *SecretReconciler) Reconcile(ctx context.Context,
 	request ctrl.Request) (result reconcile.Result, reconcileErr error) {
-	r.log = r.log.WithValues(SecretController, request.NamespacedName)
```

Comment:
is this issue also present in other controllers?

https://github.com/openshift/windows-machine-config-operator/blob/b21500c103eaa5692ccebc58ee0f6792120f9c85/controllers/node_controller.go#L74



https://github.com/openshift/windows-machine-config-operator/blob/b21500c103eaa5692ccebc58ee0f6792120f9c85/controllers/registry_controller.go#L81

Reply:
ack